pkg/autoscaler/multiscaler.go

```diff
@@ -65,7 +65,7 @@ type DeciderStatus struct {
 	DesiredScale int32
 
 	// ExcessBurstCapacity is the difference between spare capacity
-	// (how many more concurrent requests the pods in the revision
+	// (how many more tokens in terms of scaling metrics the pods in the revision
```

Comment:
maybe we should generalize this a bit differently? Something like

```
(how much more load the pods in the revision deployment can take before being overloaded)
```

Reply:
Done.